pkg/vm/context.go

```diff
@@ -52,6 +53,8 @@ type Context struct {
 	ParamCount int
 	// RetCount specifies number of return values.
 	RetCount int
+	// NEF represents nef-file of the current contract.
+	NEF *nef.File
```

Comment:
But do we really need NEF here if we only care about method tokens?

Reply:
No, fixed.